### ISSUE TITLE
Corvallis Bus App Issue

### ISSUE THREAD

**Issue by NXxEpicXCreeperxX** [RikkiGibson/Corvallis-Bus-Server#50]
Hi, so couple days ago I tried using the bus app, it wasn't working at all and keeps saying failed to load, service updates also stopped working few months ago but thought it was my phone, Now the app or shortcut don't work at all for my iPhone or iPod, its Data/Cellular connection is fine because I can load other apps or transit sites, but Corvallis Bus App won't load.

Any chance this issue could be checked/looked at?
Thank you, hoping this can be fixed soon.

**Comment by RikkiGibson** [comment]
Looks like there was an SSL certificate issue. I poked some stuff in Azure and looks like it's happy again. Thank you for reporting.

> service updates also stopped working few months ago but thought it was my phone

I pushed a workaround for this. Not as nice as it was before, but it works. I've had trouble getting in touch with the city lately to have the server unblocked from accessing the city's website, so this will have to do for now.
